app/server-utils/stampy.ts

```diff
@@ -288,6 +289,7 @@ const convertToQuestion = ({name, values, updatedAt} = {} as AnswersRow): Questi
   title: name,
   pageid: extractText(values['UI ID']),
   text: renderText(extractText(values['UI ID']), values['Rich Text']),
+  markdown: extractText(values['Rich Text']),
```

Comment:
this feels icky, adding it just for this one, hidden, place. The alternative was creating a new endpoint/function which is also icky :/